doc/source/Running-at-your-site.rst

```diff
@@ -0,0 +1,155 @@
+Running at your site
+====================
+
+Installation
+------------
+
+The latest release of IMPROVER can be installed using
+`conda <https://docs.conda.io/en/latest/>`__ from the
+`conda-forge <https://anaconda.org/conda-forge/improver>`__ channel
+using:
+
+.. code:: bash
+
+   conda install -c conda-forge improver
+
+If you would like to install the development version (or install in
+development mode) you can do this via
+`setuptools <https://setuptools.readthedocs.io/en/latest/>`__ by
+downloading the code from this repository, installing the required
+dependencies and then using either ``pip install`` (``pip install -e``)
+or ``python setup.py install`` (``python setup.py develop``). Note that
+``pip install`` will not work in an empty environment due to problems
+with installation of the dependency ``iris`` via pip.
+
+Required dependencies are listed in
+```environment_py36_iris22.yml`` <https://github.com/metoppv/improver/blob/master/envs/environment_py36_iris22.yml>`__
+and
+```environment_py37_iris24.yml`` <https://github.com/metoppv/improver/blob/master/envs/environment_py37_iris24.yml>`__,
```

Comment:
These display quite unpleasantly:
https://bens-improver-fork.readthedocs.io/en/wikitest/Running-at-your-site.html

Reply:
Unfortunately, it seems not possible (or least complicated) to get "monospace" font in a hyperlink. rST and Sphinx do have some quirks. I've made the links "regular" text with the hyperlink.